app/controllers/suppliers_controller.rb

```diff
@@ -84,6 +85,14 @@ def meta_title
     "#{supplier.name} customer service performance"
   end
 
+  def meta_description
+    # rubocop:disable Layout/LineLength
+    return "Don’t switch before you compare customer services. Large energy companies ranked by number of complaints, wait times and commitments. Check your supplier." if supplier.blank?
```

Comment:
I'm wondering if this would be more readable as an `if..else`:

```ruby
if supplier.present?
  "Customer service contact details and scores for #{supplier.name}. Find out how well #{supplier.name} customer service performs and how to contact them."
else
  "Don’t switch before you compare customer services. Large energy companies ranked by number of complaints, wait times and commitments. Check your supplier."
end
```